### PR TITLE
Python 3.x

### PR DESCRIPTION
**Pull request by xze**
## Changes brought in:
1. Switch to Python 3.x
2. Giving up on using `supervisord` and running via `gunicorn` instead
## Impacted files:
- Dockerfile
- tox.ini (removed `py27` environment)

---

This change is [Reviewable](https://reviewable.io/reviews/eclecticiq/opentaxii/49)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = flake8,py27,py34
+envlist = flake8,py34
```
Comment (traut) [inline tox.ini]: I would prefer to have both environments - py27 and py34 - running tests. No reason to focus only on py34 just now

File: Dockerfile [hunk 2]
```diff
@@ -35,8 +32,6 @@ VOLUME [ "/data", "/input" ]
 COPY docker/entrypoint.sh /entrypoint.sh
 ENTRYPOINT [ "/entrypoint.sh" ]
 
-# Expose and Run using supervisor
-COPY docker/supervisord.conf /supervisord.conf
+# Expose and Run in gunicorn
```
Comment (traut) [inline Dockerfile]: why have you decided to drop "supervisord-gunicorn" combination and run server just in gunicorn?

**Comment by aukjan** [comment]
@xze @traut I think that we should have: 
* Support for both Python 2 and Python 3
* Have docker containers using both. So actually having two different docker files. 

On supervisord, I think it makes sense to remove, as docker itself is the thing that you run, and supervisord is not a must.

**Comment by aukjan** [comment]
@xze @traut  what about this one?

**Comment by traut** [comment]
@aukjan it is still in limbo

**Comment by aukjan** [comment]
can we then move on this?  can you two work together?

**Comment by traut** [comment]
@aukjan pull request needs to be adjusted. I guess the question is about priorities. @xze has other stuff on his plate

**Pull request closed without merging**